test/Microsoft.AspNetCore.Mvc.IntegrationTests/MutableObjectModelBinderIntegrationTest.cs

```diff
@@ -1355,13 +1344,10 @@ private class Person9
             Assert.NotNull(model.Customer.Address);
             Assert.Equal(AddressStreetContent, model.Customer.Address.Street);
 
-            Assert.Equal(1, modelState.Count);
+            Assert.Equal(0, modelState.Count);
```

Comment:
These are all over the test code. It's not wrong. I'd suggest not cleaning it up.